### DISCUSSION TITLE
Get total worker count in wdio config file

### DISCUSSION THREAD

**Discussion by Pandicool** [webdriverio/webdriverio#8308] [Q&A]
Hi team,

Is there a way to get total execution worker count in wdio config file

**Comment by christian-bromann** [comment]
You mean the amount of worker run totally? I don't think so. What is your use case?

**Comment by Pandicool** [comment]
i am using bail out logic so after bailing out at certain point. from oncomplete method i can able to get the passed and failed test case count but not able to get the total test cases.

![image](https://user-images.githubusercontent.com/51842349/167175313-60aa4ed1-a06e-48cc-ac98-ba8a9692a35f.png)

**Comment by Pandicool** [comment]
is there any another way we can get the total count or else aborted test cases count due to bail for prepare a report.
![image](https://user-images.githubusercontent.com/51842349/167175625-7ffd2bb0-053a-4dc6-8dce-0b90bce50d8f.png)
i am preparing the report like this

**Comment by christian-bromann** [comment]
I think we don't populate that information anywhere. The [launcher](https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-cli/src/launcher.ts#L208) has that information though.

**Reply by Pandicool** [reply]
yes, is there any other way to get that data. i saw the data present in [here](https://github.com/webdriverio/webdriverio/blob/a23f474e14db5e584ef7d57129e4ec34c32e0974/packages/wdio-cli/src/interface.ts#L64). i want the same data in wdio-config. please help me to get a data there

**Reply by christian-bromann** [reply]
> i want the same data in wdio-config.

What do you mean by that?

**Reply by Pandicool** [reply]
I just need to know how to get the total number of  execution test case count (total worker count )in config file or current execution object

**Reply by christian-bromann** [reply]
As I mentioned there is currently no way to get this number in one of the hooks of your config file. Feel free to suggest ways how the data could be exposed to users.

**Reply by Pandicool** [reply]
@christian-bromann 
so i see some data's like passed and failed count has been exported from [interface](https://github.com/webdriverio/webdriverio/blob/dda959a0e86948104417b773ced5804e29549c55/packages/wdio-cli/src/interface.ts#L29)
and [launcher](https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-cli/src/launcher.ts) so definitely there will be a way to get the total count also.

why because i am trying to add the bail out logic and its perfectly working fine but trying to make it as a report format for that we need that total executor count.

![image](https://user-images.githubusercontent.com/51842349/168119971-359d6db3-6342-4ca1-b59f-8f78ea1ca89f.png)
![image](https://user-images.githubusercontent.com/51842349/168120083-1097b01d-18b1-4914-8940-bd6f67333653.png)

total executor has 12 cases. as per bail out logic after 2 failure execution stopped but for making the proper bailout report we need that total execution worker count

**Comment by Pandicool** [comment]
@christian-bromann can you suggest work around to get the [launcher](https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-cli/src/launcher.ts#L208) information.
